header-rewrite-filter/header_processor.cc

```diff
@@ -95,6 +94,71 @@ namespace HeaderRewriteFilter {
         request_headers->setPath(request_path); // should never return an error
     }
 
+
+    void SetBoolProcessor::setStringsToCompare(std::pair<absl::string_view, absl::string_view> strings_to_compare) {
+        std::string first_string = std::string(strings_to_compare.first);
```

Comment:
Nit: these might cause an unnecessary copy, in the `std::string(` and then the assignment.
`std::string first_string(strings_to_compare.first);` should only be 1 copy instead of 2.